setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     requests>=2.22
     scipy>=1.1.0
     spherical-geometry>=1.2.18
-    stdatamodels~=0.1.0
+    stdatamodels
```

Comment:
stdatamodels 0.2.0 requires https://github.com/spacetelescope/jwst/pull/5512, which we'll hopefully merge today.  I'd like to require `<1.0` here and also for stpipe, in case we need to make breaking changes for romancal once jwst is frozen.